ably-ios/ARTPresenceMap.h

```diff
@@ -7,22 +7,24 @@
 //
 
 #import <Foundation/Foundation.h>
+#import "CompatibilityMacros.h"
 
 @class ARTPresenceMessage;
 
+ART_ASSUME_NONNULL_BEGIN
+
 /// Used to maintain a list of members present on a channel
-@interface ARTPresenceMap : NSObject {
-    
-}
+@interface ARTPresenceMap : NSObject
+
+/// List of members.
+/// The key is the clientId and the value is the latest relevant ARTPresenceMessage for that clientId.
+@property (readonly, atomic, getter=getMembers) __GENERIC(NSDictionary, NSString *, ARTPresenceMessage *) *members;
```

Comment:
The presence API is pretty broken anyway already. I see that you're only adding generics, which I think is OK. I'll get the API right.